templates/docs/patterns/icons.md

```diff
@@ -7,6 +7,29 @@ context:
 
 # Icons
 
+{% macro icon_section(icon_set, theme) -%}
```

Comment:
Nice! Thanks for taking time to refactor it.

One thing I'd suggest is moving it above the `#Icons` header to have clearer distinction between content and "meta" definitions.

Reply:
good shout - will do

Reply:
Please do ;)

Reply:
Rargh, rebase/cherry picking snafu...